CenterForOpenScience/osf.io
# [Hotfix][ENG-3678] Configurable Preprint Metrics start date

## Purpose
Avoid pitfalls relating to old unvalidatable metrics data for preprints.

## Changes
- Subclass MetricsViewMixin to set a default `after` when requesting `total` period of view/download metrics
- Use in Preprint/Provider views that allow metrics data embeds

## QA Notes
- Test environments are unlikely to have data old enough verify that it works as intended, but should have enough to verify that it does not break.

## Documentation
Docstring added, otherwise N/A

## Side Effects
None expected

## Ticket
[ENG-3678](https://openscience.atlassian.net/browse/ENG-3678)
